bamboo-specs/bamboo.yml

```diff
@@ -0,0 +1,257 @@
+---
+version: 2
+plan:
+  project-key: ADDRHIER
+  key: AHML
+  name: Address Hierarchy Module Latest
+stages:
+- Build and Test:
+    manual: false
+    final: false
+    jobs:
+    - Build and Test JDK 21
```

Comment:
How about the other Java versions?

Reply:
Can we add them and then we deal with the test failures?

Reply:
Alright.